src/sdl_ui.cpp

```diff
@@ -16,35 +16,42 @@
  */
 
 #include "system.h"
+
 #ifdef USE_SDL
 
-// Headers
 #include "sdl_ui.h"
+
 #ifdef _WIN32
-	#define WIN32_LEAN_AND_MEAN
-	#ifndef NOMINMAX
-	#define NOMINMAX
-	#endif
-	#include <windows.h>
-	#include "SDL_syswm.h"
+#  define WIN32_LEAN_AND_MEAN
+#  ifndef NOMINMAX
+#    define NOMINMAX
+#  endif
+#  include <windows.h>
+#  include "SDL_syswm.h"
```

Comment:
Some years ago we changed these to the way I did because of http://stackoverflow.com/a/789156/1244033